### ISSUE TITLE
Component won't install on React 17.

### ISSUE THREAD

**Issue by bartdorsey** [strateos/react-map-interaction#83]
It looks like when you set the "minimum" version to 16.3.0 you did it incorrectly.   `^16.3.0` means any version of 16... which means react 17 doesn't count and so you get an error trying to install:

```% npm install react-map-interaction                                                                                         21-03-22 - 15:48:33
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: starter@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.3.0" from react-map-interaction@2.0.0
npm ERR! node_modules/react-map-interaction
npm ERR!   react-map-interaction@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/echo/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/echo/.npm/_logs/2021-03-22T20_48_36_892Z-debug.log```

**Comment by bartdorsey** [comment]
You need `>=16.3.0`

**Comment by scottyantipa** [comment]
Yep I’ll get this done

**Comment by scottyantipa** [comment]
Please take a look at the PR https://github.com/strateos/react-map-interaction/pull/102

**Comment by scottyantipa** [comment]
This is available in 2.1.0
